beerus_cli/src/runner.rs

```diff
@@ -186,6 +186,19 @@ pub async fn run(beerus: BeerusLightClient, cli: Cli) -> Result<CommandResponse>
                     .await
             }
             StarkNetSubCommands::QuerySyncing {} => starknet::query_starknet_syncing(beerus).await,
+            StarkNetSubCommands::QueryEstimateFee {
+                block_id,
+                block_id_type,
+                broadcasted_transaction,
+            } => {
+                starknet::query_starknet_estimate_fee(
+                    beerus,
+                    block_id.to_string(),
+                    block_id_type.to_string(),
+                    broadcasted_transaction.to_string(),
+                )
+                .await
+            }
```

Comment:
This is what this call looks like

```bash
source .env && ./target/release/beerus_cli starknet query-estimate-fee \
--block-id-type tag \
--block-id latest \
--broadcasted-transaction "{\"type\":\"INVOKE\",\"max_fee\":\"0x0\",\"version\":\"0x1\",\"signature\":[\"0x156a781f12e8743bd07e20a4484154fd0baccee95d9ea791c121c916ad44ee0\",\"0x7228267473c670cbb86a644f8696973db978c51acde19431d3f1f8f100794c6\"],\"nonce\":\"0x0\",\"sender_address\":\"0x5b5e9f6f6fb7d2647d81a8b2c2b99cbc9cc9d03d705576d7061812324dca5c0\",\"calldata\":[\"0x1\",\"0x7394cbe418daa16e42b87ba67372d4ab4a5df0b05c6e554d158458ce245bc10\",\"0x2f0b3c5710379609eb5495f1ecd348cb28167711b73609fe565a72734550354\",\"0x0\",\"0x3\",\"0x3\",\"0x5b5e9f6f6fb7d2647d81a8b2c2b99cbc9cc9d03d705576d7061812324dca5c0\",\"0x3635c9adc5dea00000\",\"0x0\"]}"
```

This is the output

```
FeeEstimate { gas_consumed: 5194, gas_price: 33106896397, overall_fee: 171957219886018 }
```

Reply:
tested looks great